website/content/docs/install-boundary/initialize.mdx

```diff
@@ -480,7 +447,7 @@ resource "boundary_role" "project_admin" {
 
 ### Log in as your new user
 
-```shell-session
+```bash
 boundary authenticate password \
   -auth-method-id <auth_method_id>
 ```
```

Comment:
I would probably keep this as `shell-session` instead of bash, since it's not a bash script. We tend to use shell-session for all generic executable code, unless it's bash-specific